packages/odyssey-react-mui/src/OdysseyTranslationProvider.tsx

```diff
@@ -44,14 +44,33 @@ export type OdysseyTranslationProviderProps<
   translationOverrides?: TranslationOverrides<SupportedLanguages>;
 };
 
+const formatLanguageCodeToHyphenated = <SupportedLanguages extends string>(
+  languageCode: OdysseyTranslationProviderProps<SupportedLanguages>["languageCode"]
+) => languageCode?.replace(/_/g, "-");
+
 export const OdysseyTranslationProvider = <SupportedLanguages extends string>({
   children,
   languageCode,
   translationOverrides,
 }: OdysseyTranslationProviderProps<SupportedLanguages>) => {
   useEffect(() => {
+    const languageCodeIncludesUnderscore = languageCode?.includes("_");
+
+    const normalizedLanguageCode = languageCodeIncludesUnderscore
+      ? formatLanguageCodeToHyphenated<SupportedLanguages>(languageCode)
```

Comment:
Do we need to check if the provided `languageCode` is in our list of supported languages?

Reply:
I'm not sure. There have been some changes here recently that I don't have full context on. @KevinGhadyani-Okta With the recent changes to support BYOL, we really can't check that the `languageCode` exists in out supported languages list can we?

Reply:
I think passing the `SupportedLanguages` and ensuring it matches the provider props definition for `languageCode` covers this for us